events/src/test/java/com/redhat/runtimes/inventory/events/ArchiveAnnouncementParserTest.java

```diff
@@ -17,4 +17,13 @@ public void simpleParse() throws IOException {
     var announce = parser.fromJsonString(json);
     assertEquals("12345", announce.getOrgId());
   }
+
+  @Test
+  public void eggDateTime() throws IOException {
+    var json = readFromResources("egg_datetime.json");
+    var parser = new ArchiveAnnouncementParser();
+
+    var announce = parser.fromJsonString(json);
+    assertEquals("1.0.0", announce.getVersion());
```

Comment:
shouldn't we check the timestamp, (what is the subject of the change?)

Reply:
We should - but this is trickier than it might appear. We can fix this later.